src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -139,6 +147,15 @@ int main(int argc, const char *argv[]) {
 
   EmscriptenGlueGenerator generator(wasm);
   generator.fixInvokeFunctionNames();
```

Comment:
That's mysterious, what does asm2wasm do with i64 invokes?
LGTM otherwise

Reply:
asm2wasm also does `passRunner.add("legalize-js-interface");`, like this PR adds.

I think fixInvokeFunctionNames does something else, it connects to the right invoke names (I think the backend emits other names?), but that's separate from legalizing their types.

I think it's ok to do that first, then legalization (the order in this PR), as legalization doesn't change the names of functions.